packages/terra-core-docs/src/terra-dev-site/doc/status-view/example/Custom.jsx

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
-import IconDiamond from 'terra-icon/lib/icon/IconDiamond';
+import IconDueSoon from 'terra-icon/lib/icon/IconDueSoon';
 import StatusView from 'terra-status-view';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 
 const StatusViewButtons = [
   {
-    text: 'Action 1',
+    'aria-describedby': 'submittimesheet',
+    id: 'Submit',
     key: 1,
+    text: 'Submit',
+    variant: 'emphasis',
   }, {
-    text: 'Action 2',
+    'aria-describedby': 'canceltimesheet',
+    id: 'Cancel',
     key: 2,
+    text: 'Cancel',
   },
 ];
 
 const Custom = () => (
-  <StatusView
-    message="Custom Message"
-    customGlyph={<IconDiamond />}
-    title="Custom Title"
-    buttonAttrs={StatusViewButtons}
-  />
+  <div>
+    <VisuallyHiddenText aria-hidden id="submittimesheet" text="Select to Submit Timesheet" />
+    <VisuallyHiddenText aria-hidden id="canceltimesheet" text="Select to Cancel Timesheet" />
+    <StatusView
+      message="Once submitted cannot be revised"
+      customGlyph={<IconDueSoon />}
+      title="Timesheet"
+      buttonAttrs={StatusViewButtons}
+    />
+    <p>
+      Note: Action button labels should provide proper message or title. For Accessibility best practice use the ARIA attributes like aria-label to provide the context.
```

Comment:
nit: slight rewording
```suggestion
      Note: Action button labels should provide a proper message or title. For accessibility best practice use ARIA attributes (such as aria-label) to provide context.
```

Reply:
will do this change. Thanks